argoexec/requirements-sanity.in

```diff
@@ -0,0 +1,5 @@
+charmed_kubeflow_chisme
+juju<4.0
```

Comment:
I remember commenting before, I don't think python-libjuju, ops and the pytest-operator are required for this test.

Reply:
Removed.